miniconf-py/README.md

```diff
@@ -0,0 +1,6 @@
+# Miniconf Python Utility
+
+This directory contains a Python package for interacting with Miniconf utilities.
+
+## Installation
+Run `pip install .` from this directory to install the `miniconf` package.
```

Comment:
Is there a way to make this `pip install git+https://...` compatible? That would make the installation nicer.
We might just want to split the repo.

Reply:
I would assume it already is? You should be able to do `pip install git+https://github.com/quartiq/miniconf@feature/python-utility#subdirectory=miniconf-py` (with the absence of branch once merged) as well (once I fix the build)

Reply:
Added a note to the README on how to do this.